multinet/api/views/common.py

```diff
@@ -42,6 +42,31 @@ def paginate_queryset(self, query: ArangoQuery, request: Request) -> List[Dict]:
 
 
 class WorkspaceChildMixin(NestedViewSetMixin):
+    prefix = None
+
+    @property
+    def workspace_field(self):
+        field = 'workspace__name'
+        print(type(self))
```

Comment:
We don't generally have print statements in the code. Should we remove this one?

Reply:
Absolutely.

```suggestion

```